src/Definition.php

```diff
@@ -95,4 +96,31 @@ class Definition
      * @var string
      */
     public $documentation;
+
+    /**
+     * Gets the definitons of all ancestor classes
+     *
+     * @return Definition[]
```

Comment:
phpDocumentor has no support for generic types, but you can describe it after the type